cmd/adapter-rest/startcmd/start.go

```diff
@@ -1116,3 +1137,22 @@ func getPresentationExchangeProvider(configFile string) (*presentationex.Provide
 
 	return p, nil
 }
+
+func readOutputDescriptorFile(outputDescriptorsFile string) (outputDescriptors []*cm.OutputDescriptor, err error) {
+	credentialManifestBytes, err := ioutil.ReadFile(filepath.Clean(outputDescriptorsFile))
+	if err != nil {
+		return nil, fmt.Errorf("read output descriptors file : %w", err)
+	}
+
+	err = json.Unmarshal(credentialManifestBytes, &outputDescriptors)
+	if err != nil {
+		return nil, fmt.Errorf("failed to unmarshal output descriptor file: %w", err)
+	}
+
+	err = cm.Validate(outputDescriptors)
```

Comment:
should loop on map and validated value (`[]OutputDescriptors`) in each iteration

Reply:
done